src/drivers/distance_sensor/lightware_sf45_serial/module.yaml

```diff
@@ -0,0 +1,63 @@
+module_name: Lightware SF45 Rangefinder (serial)
+serial_config:
+    - command: lightware_sf45_serial start -R 0 -d ${SERIAL_DEV}
+      port_config_param:
+        name: SENS_EN_SF45_CFG
+        group: Sensors
+        default: TEL2
+      num_instances: 1
+      supports_networking: false
+
+parameters:
+    - group: Sensors
+      definitions:
+          SF45_UPDATE_CFG:
+              description:
+                  short: Update rate in Hz
+                  long: |
+                      The SF45 sets the update rate in Hz to allow greater resolution
+              type: enum
+              values:
+                  1: 50hz
+                  2: 100hz
+                  3: 200hz
+                  4: 400hz
+                  5: 500hz
+                  6: 625hz
```

Comment:
do the high rates actually work?

Reply:
I’ve been testing at 100. I am not sure what is feasible with higher rates as the uart cable the sensor comes with is long and I have not implemented anything to alleviate malformed packet errors due to noise/ground loops. Right now we weed out these bad packets with the checksum. I’m sure someone running a nice rs422 setup or over CAN can mitigate this and get good data at higher rates…

Reply:
It's probably better to drop the higher rates until verified. Certain types of people can't resist the urge to select the bigger number even if it's actually worse.